app/models/parliament-submission-activity.js

```diff
@@ -4,7 +4,7 @@ export default class ParliamentSubmissionActivity extends Model {
   @attr('date') startDate;
   @attr('date') endDate;
 
-  @belongsTo('user', { async: true }) submitter;
+  @belongsTo('user', { inverse: null, async: true }) submitter;
 
-  @hasMany('piece', { async: true }) pieces;
+  @hasMany('submitted-piece', { inverse: null, async: true }) submittedPieces;
```

Comment:
Just a question, since I know this was a point of contention while creating the story. We did land 100% on not persisting the `comment` that goes with the `submission-activity`?

Reply:
No idea, that was said as such in the story
> Wij hoeven (voorlopig toch) dat veld NIET bij te houden in onze data. M.a.w. de inhoud van het opmerkingen veld wordt meegestuurd naar de vlaams-parlement-sync-service bij indiening, maar daarna niet bijgehouden in het datamodel.
and I didn't know who to follow up with about it so I just implemented it as stated.

Reply:
so as discussed in standup, I'm adding the comment part to the model anyway, since it was decided while I was away.